### DISCUSSION TITLE
Hi，0.8V can't run.  error: 无法定位程序输入点CreateDXGIFactory2于动态链接库dxgi.dll上。

### DISCUSSION THREAD

**Discussion by nottellyou** [dail8859/NotepadNext#656] [General]
OS:  win7_x64
version:NotepadNext-v0.8-win64.zip
error: 
![image](https://github.com/user-attachments/assets/d64cb94a-af85-43a2-a6ba-ef1f2b497c7f)

How to deal this error?

**Comment by dail8859** [comment]
See https://github.com/dail8859/NotepadNext/issues/530#issuecomment-2027871784

The short answer is Notepad Next by default uses Qt 6.5 which does not support Windows 7.

**Comment by nottellyou** [comment]
Thinks
